main.py

```diff
@@ -152,10 +152,17 @@ def check_configurations(args, pretrain_tasks, target_tasks):
 
     assert_for_log(
         args.transfer_paradigm in ["finetune", "frozen"],
-        "Transfer paradigm %s not supported!" % args.transfer_paradigm,
+        "Transfer paradigm %s not supported." % args.transfer_paradigm,
     )
 
     if args.do_pretrain:
+        task_directory, _, _ = check_for_previous_checkpoints(
```

Comment:
task_directory still sounds like a path name. Could you use a name that says what this variable is doing here?